Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM ruby:3.3.0
 
 # Install packages
-RUN apt-get update && apt-get install -y build-essential nodejs libpq-dev npm fop=1:2.* libsaxon-java libsaxonb-java chromium
+RUN apt-get update && apt-get install -y build-essential nodejs libpq-dev npm fop=1:2.* libsaxon-java libsaxonb-java chromium \
+    && apt-get -y install libx11-xcb1 libxcomposite1 libasound2 libatk1.0-0 libatk-bridge2.0-0 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6
```

Comment:
Este som sem dolozila podla https://ploi.io/documentation/server/how-to-install-puppeteer-on-ubuntu, nestacilo len chromium. Pozri prosim @celuchmarek, ci sme s tym ok.
Uz je to co potrebujeme funkcne takto.

Reply:
Toto moze celkom nafuknut ten image ale teda skusme co to spravi.

Reply:
čo už, aj tak má ten image už cez 3GB, takže už je to jedno.